webapp/src/main/webapp/schedule-news-drawer/components/ExoScheduleNewsDrawer.vue

```diff
@@ -0,0 +1,211 @@
+            <div class="grey--text my-4 scheduleInfoCursor">{{ $t('news.composer.choosePostDate') }}</div>
+            <div class="d-flex flex-row flex-grow-1">
+              <slot name="postDate"></slot>
+              <date-picker
+                v-model="postDate"
+                :min-value="minimumPostDate"
+                class="scheduleDatePicker flex-grow-1 my-auto" />
+              <div class="d-flex flex-row flex-grow-0">
+                <slot name="postDateDateTime"></slot>
+                <time-picker
+                  v-model="postDateTime"
+                  :min="minimumPostDateTime"
+                  class="me-4" />
+              </div>
+            </div>
+          </div>
+          <v-radio
+            v-if="allowNotPost"
+            :label="$t('news.composer.notPost')"
+            value="notPost"
+            class="postModeText mt-4"
+            @click="changeDisable">
+            <span slot="label" class="postModeText">{{ $t('news.composer.notPost') }}</span>
+          </v-radio>
+          <div v-if="allowNotPost && postArticleMode!=='later' && postArticleMode !=='immediate'" class="grey--text my-4 ml-4 scheduleInfoCursor">{{ $t('news.composer.chooseNotPost') }}</div>
+        </v-radio-group>
+      </template>
+      <template slot="footer">
+        <div v-if="editScheduledNews !== 'editScheduledNews'" class="d-flex justify-end">
+          <v-btn
+            :disabled="disabled"
+            class="btn btn-primary ms-2"
+            @click="selectPostMode">
+            {{ saveButtonLabel }}
+          </v-btn>
+        </div>
+        <div v-else class="d-flex justify-end">
+          <v-btn
+            class="btn ms-2"
+            @click="closeDrawer">
+            {{ $t('news.composer.cancel') }}
+          </v-btn>
+          <v-btn
+            :disabled="disabled"
+            class="btn btn-primary ms-2"
+            @click="selectPostMode">
+            {{ $t('news.composer.confirm') }}
+          </v-btn>
+        </div>
+      </template>
+    </exo-drawer>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    newsId: {
+      type: String,
+      required: false,
+      default: null
+    },
+  },
+  data: () => ({
+    drawer: false,
+    disabled: false,
+    postArticleMode: 'later',
+    postDateTime: '8:00',
+    allowPostingLater: false,
+    editScheduledNews: false,
+    allowNotPost: false,
+    schedulePostDate: null,
+    postDate: null,
+  }),
+  watch: {
+    postDate(newVal, oldVal) {
+      if (!this.postDate || !newVal || !oldVal || new Date(newVal).getTime() === new Date(oldVal).getTime()) {
+        return;
+      }
+      const postDate = new Date(this.postDate);
+      const scheduleDate = new Date(this.schedulePostDate);
+      if (postDate.getTime() !== scheduleDate.getTime()){
+        this.disabled = false;
+      } else {
+        this.disabled = true;
+      }
```

Comment:
Could be simplified by
```suggestion
        this.disabled = postDate.getTime() === scheduleDate.getTime();
```